tests/security/apparmor_profile/usr_sbin_smbd.pm

```diff
@@ -56,7 +56,7 @@ sub samba_server_setup {
     systemctl("restart smb");
 
     select_console 'x11';
-    y2_module_guitest::launch_yast2_module_x11(module => "samba-server", target_match => "samba-server-installation", match_timeout => 200);
+    launch_yast2_module_x11(module => "samba-server", target_match => "samba-server-installation", match_timeout => 200);
```

Comment:
I wonder how that one worked before, but I don't think that we need to change anything here or we need at least to add an import, as we inherit from apparmor test: `use base apparmortest;`

Reply:
I also was surprised that somewhere it was used differently (here and in yast2_control_center), but I started VRs for that. If they will fails, I'll revert that back.

Reply:
Perl is a mistery sometimes, but I bet that this time it's going to fail finding this method, as in all other cases (I've checked all of them), we either have explicit import or inherit from `y2_module_guitest`. Anyway, let's see if that fails, if not I will have to explain it to myself =)

Reply:
Yes, I guess it is because the method is imported in the parent module. It failed, so I reverted it back.